docs/install.md

```diff
@@ -0,0 +1,31 @@
+Installing the Pelican Platform
+===============================
+
+This document explains how a user can download and install the Pelican client. 
+
+Installation Steps:
+--------------------
+
+### Install the Pelican Platform Binary
+Navigate to the [Pelican release page](https://github.com/PelicanPlatform/pelican/releases/). Download the proper binary for the system you are running on and select which version you would like to install. 
+
+Our versions are built like so:
+For example: 7.1.2
+- 7 represents the latest major version release
+- 1 represents feature releases
+- 2 represents a bug fix/patch release
+
+### Extract the Binary
+Once the package has finished downloading, place it in your workspace and extract the binary
```

Comment:
This comment is really only applicable to the `.tar.gz` files we release, and this step might be unclear to people. Can you offer some terminal commands that the average person with basic CLI experience could follow for extracting the binary?

Once done with that, we should also provide information for installing the `apk`, `rpm` and `deb` files on their respective systems. I think we're okay not getting into too much detail to explain how to build from source.